lib/hublot/pretty.rb

```diff
@@ -16,20 +16,27 @@ def pretty(clock=Time.now)
       "Sunday" => 7
     }
 
+    # Strip out duplicate spaces that may have been mistakes by the engine (this happens sometimes)
+    pretty_str.gsub('  ', ' ')
```

Comment:
I recommend we change this to capture any repeating space and turn it into one space. Something like `pretty_str.gsub(/\s+/, ' ')`